inventory/group_vars/us.yml

```diff
@@ -4,7 +4,7 @@ checkout_zone: USA
 country_code: US
 currency: USD
 locale: en_US
-available_locales: en_US
+available_locales: en_US, es
```

Comment:
I keep that `es` but in the future, we may want to use `es_US` instead.

Reply:
@andrewpbrett @sauloperez I would do right away an es_US : remember that the main menu is made with translations. So the about link in spanish could never go to the USwebsitee and would always link to Katuma's.
We have this in FR right now with the IT translation and it's pretty annoying.